portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DefinitionsSearch.testcase

```diff
@@ -79,4 +88,83 @@ definition {
 		ACDefinitions.noViewAddSearchQueryButton(query = "Test Query 5");
 	}
 
+	@description = "Feature ID: LRAC-13934 | Automation ID: LRAC-14680 | Test Summary: Customized query string will still collect search term"
+	@priority = 4
+	test ViewKeywordSearchedNewStringParameter {
```

Comment:
Hey @joshitagaki, I think you forgot to add this test case to the test map, could you do that?

Reply:
Thanks! Added: https://docs.google.com/spreadsheets/d/1Fk3jlL25H1T0D8SPS8b83Uzk7cFP8uggGmenbDXm8ro/edit#gid=1922969374&range=D1088